source/contracts/reporting/Market.sol

```diff
@@ -166,6 +192,8 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
     function updateTentativeWinningPayoutDistributionHash(bytes32 _payoutDistributionHash) public returns (bool) {
         var (_firstPlaceHash, _secondPlaceHash) = MarketExtensions(controller.lookup("MarketExtensions")).getOrderedWinningPayoutDistributionHashes(this, _payoutDistributionHash);
 
+        require(_firstPlaceHash != bytes32(0));
```

Comment:
This requirement is only when placing a dispute bond.  It is acceptable for someone to buy outcome A, B or C when the current stakes are:
A: -10,000 REP
B: 0 REP
C: -500 REP

Reply:
We should be able to assert this for all cases as we should never end up in the situation above since this applies when placing a dispite bond. We can remove this but it would break the system if we end up making changes where this is no longer the case

Reply:
Actually, I suppose since dispute bonds are the only thing that _decreases_ a staked outcome, this will always be true after every normal report so the check _effectively_ is only on the disputes.  Feel free to leave it in that case.